lib/cover.nix

```diff
@@ -0,0 +1,130 @@
+{ lib, haskellLib, pkgs }:
+
+{ name
+, version
+, library
+, tests
+}:
+
+let
+  identifier = name + "-" + version;
+  toBashArray = arr: "(" + (lib.concatStringsSep " " arr) + ")";
+
+  testsAsList = lib.attrValues tests;
+  checks      = builtins.map (d: haskellLib.check d) testsAsList;
+
+  # Exclude test modules from tix file. The Main module is
```

Comment:
Why do we exclude test modules?

Reply:
Added a comment: https://github.com/input-output-hk/haskell.nix/pull/762/files#diff-4bba04d96602cc233c6b24d553f66222R16